stdlib/2/formatter.pyi

```diff
@@ -0,0 +1,104 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/formatter.py
+from typing import Any, IO, List, Optional, Tuple
+
+AS_IS = None
+_FontType = Tuple[str, bool, bool, bool]
+_StylesType = Tuple[Any, ...]
+
+class NullFormatter:
+    writer = ...  # type: Optional['NullWriter']
+    def __init__(self, writer: Optional['NullWriter'] = ...) -> None: ...
+    def end_paragraph(self, blankline: int) -> None: ...
+    def add_line_break(self) -> None: ...
+    def add_hor_rule(self, *args, **kw) -> None: ...
+    def add_label_data(self, format, counter: int, blankline: Optional[int] = ...) -> None: ...
+    def add_flowing_data(self, data: str) -> None: ...
+    def add_literal_data(self, data: str) -> None: ...
+    def flush_softspace(self) -> None: ...
+    def push_alignment(self, align: Optional[str]) -> None: ...
+    def pop_alignment(self) -> None: ...
+    def push_font(self, x: _FontType) -> None: ...
+    def pop_font(self) -> None: ...
+    def push_margin(self, margin: int) -> None: ...
+    def pop_margin(self) -> None: ...
+    def set_spacing(self, spacing: Optional[str]) -> None: ...
+    def push_style(self, *styles: _StylesType) -> None: ...
+    def pop_style(self, n: int = 1) -> None: ...
```

Comment:
Can you change these defaults to `...` too (there's a few more 1s and a 72 below).